portfolio/src/main/webapp/js/script.js

```diff
@@ -53,9 +53,39 @@ async function getText() {
   }
 }
 
+/**
+ * Display text from data Servlet
+ */
+async function getText() {
+  var numComments = document.getElementById("number-comments");
+  numComments = numComments.options[numComments.selectedIndex].value;
+  getNumComments(numComments);
+}
+
 /** Creates an <li> element containing text. */
-function createListElement(text) {
+function createListElement(message) {
   const liElement = document.createElement('li');
-  liElement.innerText = text;
+  liElement.className = 'message';
+  liElement.innerText = message.text;
+  liElement.setAttribute("class", "list-group-item");
```

Comment:
be consistent with your quotes. In this case, Google style guide calls for single quotes: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes